Add `slim` versions for `python` (#9)
sidestream-tech/dockerfiles: README.md

Repository for a collection of `Dockerfile`s that have helped us during development and during projects. Useful for local development, deployments and CI/CD pipelines.

Currently, we offer:

> - `python-poetry`: Where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We wanted to change that, as we believe in `poetry` as the future of python package management and thats why are we using [`acidrain`'s](https://hub.docker.com/u/acidrain) `Dockerfile`s [`python-poetry`](https://hub.docker.com/r/acidrain/python-poetry). They are very good 🙂

Reviewer: ```suggestion
- `python-poetry`: Where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We recommend to use [`acidrain`'s](https://hub.docker.com/u/acidrain) `Dockerfile`s [`python-poetry`](https://hub.docker.com/r/acidrain/python-poetry). They are very good 🙂
```
